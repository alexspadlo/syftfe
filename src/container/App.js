import React from 'react'
import { BrowserRouter as Router, Switch, Redirect, Route } from 'react-router-dom'
import Board from '../components/Board'

const App = () => {
    return (
        <Router>
            <Switch>
                <Route exact path='/syftfe' component={Board} />
                <Redirect from="*" to="/syftfe" />
            </Switch>
        </Router>
    )
}

export default App