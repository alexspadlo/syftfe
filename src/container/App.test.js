import React from 'react'
import { render } from '@testing-library/react'
import App from './App'
import configureStore from '../store'
import { Provider } from 'react-redux'

test('Check all the 9 cells is in the App and do a snapshot', () => {
	const { container } = render(
		<Provider store={configureStore()}>
			<App />
		</Provider>
	)
	const tikTokclass = container.querySelectorAll('.tictok')
	expect(tikTokclass).toHaveLength(9)
	expect(container.firstChild).toMatchSnapshot()
})