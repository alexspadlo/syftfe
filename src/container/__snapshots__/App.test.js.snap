// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check all the 9 cells is in the App and do a snapshot 1`] = `
<section
  class="w-100"
>
  <section
    class="w-100 containerBack"
  >
    <div
      class="container"
    >
      <div
        class="row m-0"
      >
        <div
          class="col-12 col-lg-5 m-auto customPaddingTop text-center"
        >
          <img
            alt="tictoc"
            class="img-fluid"
            src="tictoctoe.png"
          />
        </div>
        <div
          class="col-12 col-lg-7 customPaddingTop m-auto"
        >
          <div
            class="row m-0"
          >
            <div
              class="col-lg-9 col-12 m-auto"
            >
              <div
                class="row m-0"
              >
                <div
                  class="col clickText  tictok box0col0"
                />
                <div
                  class="col clickText  tictok box0col1"
                />
                <div
                  class="col clickText  tictok box0col2"
                />
              </div>
            </div>
            <div
              class="col-lg-9 col-12 m-auto"
            >
              <div
                class="row m-0"
              >
                <div
                  class="col clickText  tictok box1col0"
                />
                <div
                  class="col clickText  tictok box1col1"
                />
                <div
                  class="col clickText  tictok box1col2"
                />
              </div>
            </div>
            <div
              class="col-lg-9 col-12 m-auto"
            >
              <div
                class="row m-0"
              >
                <div
                  class="col clickText  tictok box2col0"
                />
                <div
                  class="col clickText  tictok box2col1"
                />
                <div
                  class="col clickText  tictok box2col2"
                />
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </section>
</section>
`;
